Rakefile

```diff
@@ -5,10 +5,12 @@ $:.unshift File.join(File.dirname(__FILE__), 'lib')
 require 'rubygems' unless ENV['NO_RUBYGEMS']
 
 require 'bundler'
+require 'pry-nav'
```

Comment:
nah, not really. Just figured you added it at one point for testing and forgot to take it back out. Probably fine to just leave it in since it's only the rakefile